models/env_service_model.py

```diff
@@ -1,14 +1,42 @@
+import os
+import sys
+from pathlib import Path
 from dotenv import load_dotenv
 
-load_dotenv()
-import os
+
+# <app/bin/main.py>/../../
+def app_root_path():
+    try:
+        return Path(sys.argv[0]).resolve().parents[1]
+    except:
+        return Path()

```

Comment:
What does path default to? CWD?

Reply:
I tested it out with a print at the beginning of bot startup and it defaults to `.`

```
path is
.
```

Reply:
I changed it to .cwd() to be more explicit.